### DISCUSSION TITLE
Skipping around on top solid layer

### DISCUSSION THREAD

**Discussion by MJAldridge6** [supermerill/SuperSlicer#3885] [Q&A]
Not sure what to call this, so I'll just describe it the best I can. I use 3 solid layers for the bottom on all my prints, the first two layers go down solid (from one side to the other.) The last solid layer will skip around to match the features of the fourth layer (but is still a _solid_ layer) Is there a way to get it to print like the first two layers and not match the perimeters of the next layer? I'll try to attach some images that might make things more clear. Any advice would be appreciated.

![infill settings](https://github.com/supermerill/SuperSlicer/assets/138165203/259a3437-5454-46dd-a719-adca2fb9fbb7)
![perimeter settings](https://github.com/supermerill/SuperSlicer/assets/138165203/75785b37-4b19-44f1-8961-2f933775815d)
![Slice settings](https://github.com/supermerill/SuperSlicer/assets/138165203/a1705203-1f5f-461f-bb48-07d8d28ae477)
1st Layer
![1st layer](https://github.com/supermerill/SuperSlicer/assets/138165203/daf768c7-d062-445f-9bc0-8412aaaeefd2)
2nd Layer
![2nd layer](https://github.com/supermerill/SuperSlicer/assets/138165203/f027efb8-2b7b-4b39-b1ee-ad57f44af747)
3rd Layer
![3rd layer](https://github.com/supermerill/SuperSlicer/assets/138165203/0185b4e5-e0d0-4e71-898b-e086018adc65)
4th Layer
![4th layer](https://github.com/supermerill/SuperSlicer/assets/138165203/2a8ab853-7051-4ae6-b6e0-1085248259d1)
Can the purple places print the same as the red?
![3rd layer markup](https://github.com/supermerill/SuperSlicer/assets/138165203/eb1f961b-3e9e-4242-8c51-3ac6052ca4eb)
3rd layer mid-print
![3rd layer mid print](https://github.com/supermerill/SuperSlicer/assets/138165203/4f4eeb6a-f7cc-4e7c-aaff-620c67e42751)

**Comment by neophyl** [comment]
Generally it is possible yes with some changing of settings.  TO see what they would need to be we need to see your project.  Please attach your zipped up project file (File>Save Project As) and the zip up the resulting .3mf file.  
Also can you state what version of SuSi you are using please.

**Reply by MJAldridge6** [reply]
Thanks for the reply! I made a small example file. you'll see how the 3rd layer has a small purple spot it goes back to fill instead of a solid layer. Thank you for looking into this! This would be great if I can get this to print in one solid pass.  SuSi ver. 2.4.58.5
[Example.zip](https://github.com/supermerill/SuperSlicer/files/12021872/Example.zip)

**Reply by neophyl** [reply]
Attached is a modified project.  The only setting changed is the Print Settings>Infill>Anchor Solid Infill by Xmm.  You want a value larger than the area you are trying to force.  Just use big numbers in mm as % is based on the extrusion width.

[Example_neo.zip](https://github.com/supermerill/SuperSlicer/files/12027560/Example_neo.zip)
